command/operator/raft/listpeers/operator_raft_list.go

```diff
@@ -82,8 +103,22 @@ func raftListPeers(client *api.Client, stale bool) (string, error) {
 		if s.Leader {
 			state = "leader"
 		}
-		result = append(result, fmt.Sprintf("%s\x1f%s\x1f%s\x1f%s\x1f%v\x1f%s",
-			s.Node, s.ID, s.Address, state, s.Voter, raftProtocol))
+
+		serverHealthData, ok := serverHealthDataMap[s.ID]
+		if ok {
+			trailsLeaderBy := leaderLastCommitIndex - serverHealthData.LastIndex
+			trailsLeaderByText := fmt.Sprintf("%d Commits", trailsLeaderBy)
+			if s.Leader {
+				trailsLeaderByText = "_"
```

Comment:
Prefer `-` (dash) over `_` (underscore) since a dash conveys "not applicable" or "skip" when generally seen in a table cell.